### DISCUSSION TITLE
Feature suggestion - charts in Jupyter

### DISCUSSION THREAD

**Discussion by dimitri-b** [erezsh/Preql#34] [Ideas]
Jupyter + Preql is a great combo for data exploration, it would be even greater if we can quickly graph data, not just look at its tabular form.
Databricks and others have something similar, I think they use Plotly in the back.
https://docs.databricks.com/notebooks/visualizations/index.html#plot-types

[Altair](https://altair-viz.github.io/) (Vega) is another great charting library.

**Comment by erezsh** [comment]
Yes, I agree that it would be nice!

I've been putting it off, because I wasn't sure how to approach it, but I might have an idea. I'll give it a try soon!
